models/tts/vits/vits_inference.py

```diff
@@ -120,6 +121,9 @@ def inference_for_single_utterance(
         )
         phone_id_seq = phon_id_collator.get_phone_id_sequence(self.cfg, phone_seq)
 
+        if self.cfg.preprocess.add_blank and self.cfg.train.multi_speaker_training:
```

Comment:
The condition `cfg.train.multi_speaker_training` can be removed, allowing the `intersperse` function to also be applicable to single-speaker training.

Reply:
Got it, removed.